test/helpers/utils/custom-deployer.ts

```diff
@@ -0,0 +1,18 @@
+import { Signer } from 'ethers'
+import { toHexString }  from '../../../src/utils' 
+
+export const deployContractCode = async (code: string, signer: Signer, gasLimit: number): Promise<string> => {
```

Comment:
Would be good to eventually put this into `core-utils` but fine to copy/paste here instead for now.